content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Manage billing with consumption limits
+subtitle: Learn how to set usage quotas per project with the Neon API
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2023-11-09T17:36:14.275Z'
+---
+
+When setting up your billing solution with Neon, you may want to impose some hard limits on how much storage or compute size a given project can consume. For example, you may want to cap how much usage your free tier users can consume versus pro or enterprise users. With the Neon API, you can use the `quota` key to set usage limits for a variety of consumption metrics. These limits act as thresholds after which all active computes for a project are [suspended](#suspending-active-computes). 
+
+## Metrics and quotas
+By default, Neon tracks a variety of consumption metrics at the project level. If you want to set quotas (max limits) for these metrics, you need to explicitly [configure](#configuring-quotas) them. 
+
+### Available metrics
+
+Here are the relevant metrics that you can set project-level quotas for:
+
+* `active_time_seconds`
+* `compute_time_seconds`
+* `written_data_bytes`
+* `data_transfer_bytes` 
+
+These consumption metrics represent total cumulative usage across all branches and computes in a given project, accrued so far in a given monthly billing period. Metrics are refreshed on the first day of the following month, when the new billing period starts. 
+
+Neon updates these metrics every 15 minutes but it could take up to 1 hour before they are reportable.
+
+To find the current usage level for any of these metrics, see [retrieving details about a project](#retrieving-details-about-a-project). You can read more about these metrics and how they impact billing [here](/docs/billing).
+
+### Corresponding quotas
+
+You can set quotas for these consumption metrics per project using the following Neon API URL:
+
+```bash
+/projects/{project_id}/settings/quota
+```
+
+The `quota` object includes an array of parameters used to set threshold limits. Their names generally match their corresponding metric:
+
+* `active_time_seconds` &#8212; Sets the maximum amount of wall-clock time allowed in total across all of a project's compute endpoints. This means the total elapsed time, in seconds, from start to finish for each transaction handled by the project's endpoints, accumulated so far during the current billing period.
```

Comment:
I don't think this part is true, since it elapsed time can include the time that your compute is active because there is an autosuspend delay period:
"This means the total elapsed time, in seconds, from start to finish for each transaction handled by the project's endpoints, accumulated so far during the current billing period." 

I think it's just the total amount of time your project's compute endpoints are allowed to be active.

Reply:
Ah, right. The inactive period, before autosuspend kicks in, is counted. How about:

"Sets the maximum amount of time your project's total compute endpoints are allowed to be active during the current billing period. It excludes time when computes are in an `Idle` state due to [auto-suspension](/docs/reference/glossary#auto-suspend-compute)."